templates/default/datadog.conf.erb

```diff
@@ -37,6 +37,10 @@ proxy_user: <%= node['datadog']['web_proxy']['user'] %>
 proxy_password: <%= node['datadog']['web_proxy']['password'] %>
 <% end -%>
 
+<% if node['datadog']['apm_enable'] == true %>
+  apm_enable: true
```

Comment:
This should be `apm_enabled` with a `d`. ( [code](https://github.com/DataDog/datadog-trace-agent/blob/dc580a0cd378bf0723e5af099605cf9d410a921f/config/agent.go#L196), [docs](https://app.datadoghq.com/trace/install) )